cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ArtifactHttpHandlerTest.java

```diff
@@ -36,6 +36,9 @@
 import co.cask.cdap.internal.app.runtime.artifact.ArtifactRepository;
 import co.cask.cdap.internal.app.runtime.artifact.plugin.Plugin1;
 import co.cask.cdap.internal.app.runtime.artifact.plugin.Plugin2;
+import co.cask.cdap.internal.app.runtime.artifact.plugin.Plugin3;
+import co.cask.cdap.internal.app.runtime.artifact.plugin.Plugin4;
```

Comment:
This is going to be really unclear. What are those plugin? Even it's just for testing, we should give some meaningful name.